storage/src/hashednode.rs

```diff
@@ -113,6 +112,23 @@ impl<T> Deref for ValueDigest<T> {
     }
 }
 
+impl<T: AsRef<[u8]>> ValueDigest<T> {
```

Comment:
nit: feels like this could be a PartialEq implementation

Reply:
Possibly, but that `Sha256::digest` is expensive so I'd rather not hide that under an operator.